perl/Makefile

```
@@ -19,7 +19,10 @@ export PATH := $(ROOT)/perl/bin:$(PATH)
 
 #------------------------------------------------------------------------------
 test:: $(LIBYAMLSCRIPT_SO_FQNP)
-	PERL_YAMLSCRIPT_DEVEL=1 prove -l $${TEST_VERBOSE:+'-v'} $(test)
+	$(MAKE) -C $(ROOT)/perl-alien distdir
+	(cd $(ROOT)/perl-alien/Alien-YAMLScript-* && perl Makefile.PL && make)
+	PERL5LIB=$(ROOT)/perl-alien/Alien-YAMLScript-*/blib/lib \
```

The tests here die because of the glob in the directory. I'm not sure how we can get it to do what we want, but if we set the specific version in there instead the tests do pass
```suggestion
	PERL5LIB=$(ROOT)/perl-alien/Alien-YAMLScript-0.1.21/blib/lib \
```
^ That's not what we want, but that's what I mean

---

Note also that you can do the same with
```
prove -I$(ROOT)/perl-alien/Alien-YAMLScript-0.1.21/blib/lib -l test
```
You don't need the `PERL5LIB`

---

Thanks. I'll work it out.